packages/compass-find-in-page/src/components/find-in-page-input.tsx

```diff
@@ -102,34 +103,27 @@ function FindInPageInput({
     toggleStatus();
   }, [dispatchStopFind, setSearchTerm, toggleStatus]);
 
-  const onKeyDown = useCallback(
+  const onEnter = useCallback(
     (evt: KeyboardEvent) => {
-      if (evt.key === 'Escape') {
-        onClose();
+      evt.preventDefault();
+      const searchValue = findInPageInputRef.current?.value;
+      if (!searchValue || searchValue === '') {
+        return dispatchStopFind();
       }
 
-      if (evt.key === 'Enter') {
-        evt.preventDefault();
-        const searchValue = findInPageInputRef.current?.value;
-        if (!searchValue || searchValue === '') {
-          return dispatchStopFind();
-        }
-
-        const back = evt.shiftKey;
-        dispatchFind(searchValue, !back, searching);
-      }
+      const back = evt.shiftKey;
```

Comment:
this feels a bit odd at this point, I wouldn't mix the approaches, why don't we use `useHotkeys` for the key strokes and do something like this?

``` js
const find = (evt, back) => { evt.preventDefault(), const searchValue = ... };
const findNext = useCallback(() => find(evt, false))
const findPrev = useCallback(() => find(evt, true))

useHotkeys('enter', findNext, { enableOnFormTags: ['INPUT'] });
useHotkeys('shift + enter', findPrev, { enableOnFormTags: ['INPUT'] });
```